package.json

```diff
@@ -26,7 +26,10 @@
     "docker": "yarn workspace prosemirror-jdita-example docker",
     "start:example": "yarn workspace prosemirror-jdita-example start",
     "build:example": "yarn workspace prosemirror-jdita-example build",
-    "build": "yarn workspace prosemirror-jdita build"
+    "build": "yarn workspace prosemirror-jdita build",
+    "test": "yarn workspace prosemirror-jdita test",
+    "coverage": "yarn workspace prosemirror-jdita coverage",
+    "coveralls": "yarn workspace prosemirror-jdita coveralls"
```

Comment:
The script `coveralls` doesn't exist in `prosemirror-jdita` workspace.

Reply:
I think it is handled by a yarn plugin. If you run `yarn coveralls` inside the prosemirror-jdita workspace after running `yarn coverage` I think you will see it do some stuff ;-)